MapboxNavigation/LanesView.swift

```diff
@@ -67,30 +67,22 @@ open class LanesView: UIView {
     }
     
     func update(for currentLegProgress: RouteLegProgress) {
-        guard let step = currentLegProgress.upComingStep else { return }
-        guard !currentLegProgress.userHasArrivedAtWaypoint else { return }
-        let durationRemaining = currentLegProgress.currentStepProgress.durationRemaining
-        
         clearLaneViews()
         
-        if let allLanes = step.intersections?.first?.approachLanes,
-            let usableLanes = step.intersections?.first?.usableApproachLanes,
-            durationRemaining < RouteControllerMediumAlertInterval {
-            
-            for (i, lane) in allLanes.enumerated() {
-                let laneView = laneArrowView()
-                laneView.lane = lane
-                laneView.maneuverDirection = step.maneuverDirection
-                laneView.isValid = usableLanes.contains(i as Int)
-                stackView.addArrangedSubview(laneView)
-            }
+        let step = currentLegProgress.currentStep
+        guard let firstInstruction = step.instructionsDisplayedAlongStep?.first,
```

Comment:
What if there is more than one visual instruction on the step?

Reply:
I think we need something like what we have for the currentSpokenInstruction but for visual instruction:

https://github.com/mapbox/mapbox-navigation-ios/blob/03b93adff9419361fefd55d5fc6c0a293df3f479/MapboxCoreNavigation/RouteProgress.swift#L458-L462

We have all the pieces slightly above this computed variable to create `currentVisualInstruction`.

Reply:
True. That was an oversight. I will make the necessary changes to account for multiple visual instructions.